test/steps/v2_steps.py

```diff
@@ -1,10 +1,15 @@
 import base64
 import json
 import os
+from pathlib import Path
 import urllib
 import unittest
 from datetime import datetime
+from secrets import token_bytes
```

Comment:
Introduce random bytes into txn notes field so that you can run the same app with the same args in a single atomic transaction.